qchem/pennylane_qchem/qchem/hf/__init__.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This subpackage provides the functionality to perform differentiable
+Hartree-Fock calculations.
+"""
```

Comment:
@soranjh I think you need to modify the `qml_qchem.rst` file to ensure these show up in the docs: https://github.com/PennyLaneAI/pennylane/blob/09e0992ee9bf5d885e601b043cd4b81c978b3c10/doc/code/qml_qchem.rst

Reply:
Or do you not intend for this to show up in the docs?

Reply:
I was thinking on of merging this feature branch now, before it gets even heavier, and then make further changes as direct PRs to master.